src/metabase/lib/drill_thru/compare_aggregations.cljc

```diff
@@ -0,0 +1,32 @@
+(ns metabase.lib.drill-thru.compare-aggregations
+  "Adds 1 or more offset()-based aggregations.
+
+  Entry points:
+
+  - Column header
+
+  Query transformation:
+
+  - Adds 1 or more aggregations that compare the clicked column over time."
+  (:require
+   [metabase.lib.drill-thru.common :as lib.drill-thru.common]
+   [metabase.lib.schema :as lib.schema]
+   [metabase.lib.schema.drill-thru :as lib.schema.drill-thru]
+   [metabase.util.malli :as mu]))
+
+(mu/defn compare-aggregations-drill :- [:maybe ::lib.schema.drill-thru/drill-thru.compare-aggregations]
+  "Column clicks on aggregated columns."
+  [query                  :- ::lib.schema/query
+   stage-number           :- :int
+   {:keys [column value]} :- ::lib.schema.drill-thru/context]
+  (when (and column
+             (nil? value)
+             (lib.drill-thru.common/mbql-stage? query stage-number)
+             (= (:lib/source column) :source/aggregations))
+    {:lib/type :metabase.lib.drill-thru/drill-thru
+     :type     :drill-thru/compare-aggregations
+     :column   column}))
+
+(defmethod lib.drill-thru.common/drill-thru-method :drill-thru/compare-aggregations
+  [_query _stage-number _drill & _args]
+  (throw (ex-info "Do not call drill-thru for compare-aggregations; add the aggregations directly" {})))
```

Comment:
I added this `throw` for consistency with [`combine_columns.cljc`](https://github.com/metabase/metabase/blob/0dbe2761f283f48aa66d40618c43bfb00545cc13/src/metabase/lib/drill_thru/combine_columns.cljc#L33-L35) but it causes a few [tests to fail](https://github.com/metabase/metabase/actions/runs/9352185641/job/25739724612?pr=43500).

@bshepherdson could you please update this PR so that the tests pass?
I think we need to filter this particular drill out of `lib/available-drill-thrus` result in these tests.